runtime/dart_isolate_unittests.cc

```diff
@@ -153,11 +164,10 @@ TEST_F(DartIsolateTest, CanRunDartCodeCodeSynchronously) {
 
 TEST_F(DartIsolateTest, CanRegisterNativeCallback) {
   ASSERT_FALSE(DartVMRef::IsInstanceRunning());
-  fml::AutoResetWaitableEvent latch;
   AddNativeCallback("NotifyNative",
-                    CREATE_NATIVE_ENTRY(([&latch](Dart_NativeArguments args) {
+                    CREATE_NATIVE_ENTRY(([&](Dart_NativeArguments args) {
```

Comment:
FYI you can probably just capture `[this]`, not a huge deal for tests.

Reply:
Done.